jujugui/static/gui/src/app/components/header-breadcrumb/_header-breadcrumb.scss

```diff
@@ -61,6 +61,7 @@
     line-height: $breadcrumb-line-height;
     margin-left: 20px;
     position: absolute;
+    min-width: 300px;
```

Comment:
I agree with your analysis @hatched. I've come across this multiple times. As I've been going I've been making things explicit and therefore more predictable so +1 from me.